### ISSUE TITLE
HTTPS option doesn't save to user profile

### ISSUE THREAD

**Issue by OGNeo** [primetime43/Xtream-Browser#16]
HTTPS is on by default, but is not saved when disabled.  i.e. next time the application is opened, even when loading the user login, HTTPS is enabled again.

**Comment by primetime43** [comment]
Issue fixed with https://github.com/primetime43/Desktop-IPTV-Browser/commit/e451d27b8416036f411da6bf1a97805b6608be09

Will mark as resolved once the branch is merged into main.

**Comment by primetime43** [comment]
Fixed with v3.1.0
